requirements/base.txt

```diff
@@ -20,9 +17,9 @@ pymongo==3.10.1           # via edx-opaque-keys
 python-dateutil==2.8.1    # via xblock
 pytz==2020.1              # via django, fs, xblock
 pyyaml==5.3.1             # via xblock
-six==1.15.0               # via edx-opaque-keys, fs, python-dateutil, stevedore, xblock
-stevedore==1.32.0         # via edx-opaque-keys
-typing==3.7.4.1           # via fs
+six==1.15.0               # via edx-opaque-keys, fs, python-dateutil, xblock
+sqlparse==0.3.1           # via django
+stevedore==2.0.0          # via edx-opaque-keys
```

Comment:
add a constraint for this like in edx-platform:

https://github.com/edx/edx-platform/blob/cfb53bb9bba9af69d7eac7e2ba86ed45bcde686b/requirements/constraints.txt#L97

Reply:
Added!